rfcs/0000-rlp-v2.md

```diff
@@ -0,0 +1,1380 @@
+# RFC RateLimitPolicy v2
```

Comment:
Regarding the the leading `0000` in the name of the file, since this is our most likely RFC to be merged...should this be incremental and starting with `0001` or was meant for different version of the RFC proposal ?